content/pt-br/docs/reference/glossary/userns.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Namespace do usuário
+id: userns
+date: 2021-07-13
+full_link: https://man7.org/linux/man-pages/man7/user_namespaces.7.html
+short_description: >
+  Um recurso do kernel Linux para emular privilégios de superusuário para usuários sem privilégios.
+
+aka:
+tags:
+- security
+---
+
+Um recurso do kernel para emular o root. Usado para "contêineres sem root".
+
+<!--more-->
+
+Os namespaces do usuário são um recurso do kernel Linux que permite que um usuário não root emule privilégios de superusuário ("root"), por exemplo, para executar contêineres sem ser um superusuário fora do contêiner.
+
+O namespace do usuário é eficaz para mitigar os danos de possíveis ataques fora de contêineres.
```

Comment:
"ataques fora de contêineres" does not deliver the original meaning here, which would be for an attacker to break-out, i.e., escape the container. The overall theme here is that the attack starts within the container, but then leaks out to the host where the container is running.

Some suggestions:
* "mitigar os danos de possíveis ataques de escape dos limites de contêineres"
* "mitigar os danos de um potencial ataque em que o adversário escapa dos limites do contêiner"

These docs seem to use "escape de limites de contêiner": https://docs.aws.amazon.com/pt_br/AmazonECS/latest/bestpracticesguide/security-tasks-containers.html
Whereas these docs seem to use "escape de contêiner": https://docs.microsoft.com/pt-br/virtualization/windowscontainers/manage-containers/container-security#container-security-servicing-criteria

Reply:
Thanks for the articles and the review.